### ISSUE TITLE
Fighting AI

### ISSUE THREAD

**Issue by fabbe50** [TheResimulators/Sim-U-Kraft-Resimulated-1.15#43]
So there's three options.

1: Make it so the Sim use any weapon they have in their inventory to fight off mobs that enter their vicinity.
2: Make it so the Sim can only use a specific type of weaponry (Warriors can only use Sword and Shield, Archers can only use bows, Rogues can only use crossbows and Mages can only use potions etc.) This will ofc mean that the Sims will have to make the resources for fighting first (making arrows, brewing potions etc.) Most likely there will be Sims that can't fend for themselves in this case and won't be able to wield any weapon (more than possibly a knife?). 
3. And the third option is to make this a job. Like have a Guards type job. 

I've given this a fair amount of thought and I'm rooting for the second option, but I wanna do right and ask what you think before I commit myself into doing what I personally want. 

Tell me what you think.

**Comment by Reprevise** [comment]
I do like option 2 but I feel like option 3 is the best but I'd like to make a suggestion. I like the idea of a "military" with a bunch of guards and the other sims can fight with what they have in their inventory under attack. So a mix of 3 and 1 essentially.

**Comment by general5001** [comment]
I like option 2 and 3

**Comment by fabbe50** [comment]
So like give the military some kind of buffed abilities? Cause the civilians in that case can't have the same fighting capabilities as the military, cause that would make the military useless.

**Comment by Reprevise** [comment]
Yeah, the military spends all it's time training and they get buffed and the civilians who aren't in the military can fight, but not that well. The military would fight mobs at night and we could make an invasion scenario or something but that's for a separate time

**Comment by fabbe50** [comment]
@Reprevise That does sound tempting tbh

**Comment by aidie8** [comment]
Civilians can defend them selves if they are attacked first. Guards could patrol the village and actively attack mobs entered the village

**Comment by aidie8** [comment]
When training guards we could select what type of guards we want. Like warrior, rogue, archer. Mage. We could also make it so to get a mage they have to be level 10.(if we add a leveling system)

**Comment by general5001** [comment]
I like aidie's idea

**Comment by fabbe50** [comment]
I like that idea @aidie8
